migrations/v2.js

```diff
@@ -0,0 +1,196 @@
+  whereContent('GMCQ - where GMCQ', async (content) => {
+    GMCQs = content.filter(({ _component }) => _component === 'gmcq');
+    return GMCQs.length;
+  });
+  mutateContent('GMCQ - add _recordInteraction attribute', async (content) => {
+    GMCQs.forEach(GMCQ => {
+      GMCQ._recordInteraction = true;
+    });
+    return true;
+  });
+  checkContent('GMCQ - check _recordInteraction attribute', async (content) => {
+    const isValid = GMCQs.every(GMCQ =>
+      _.has(GMCQ, '_recordInteraction')
+    );
+    if (!isValid) throw new Error('GMCQ - _recordInteraction not found');
+    return true;
+  });
+  updatePlugin('GMCQ - update to v2.0.2', { name: 'adapt-contrib-gmcq', version: '2.0.2', framework: '>=2.0.0' });
+});
+
+describe('GMCQ - v2.0.2 to v2.0.3', async () => {
+  let GMCQs;
+  whereFromPlugin('GMCQ - from v2.0.2', { name: 'adapt-contrib-gmcq', version: '<2.0.3' });
+  whereContent('GMCQ - where GMCQ', async (content) => {
+    GMCQs = content.filter(({ _component }) => _component === 'gmcq');
+    return GMCQs.length;
+  });
+  mutateContent('GMCQ - add _canShowModelAnswer attribute', async (content) => {
+    GMCQs.forEach(GMCQ => {
+      GMCQ._canShowModelAnswer = true;
+    });
+    return true;
+  });
+  mutateContent('GMCQ - add _columns attribute', async (content) => {
+    GMCQs.forEach(GMCQ => {
+      GMCQ._columns = 0;
+    });
+    return true;
+  });
+  checkContent('GMCQ - check _canShowModelAnswer attribute', async (content) => {
+    const isValid = GMCQs.every(GMCQ =>
+      _.has(GMCQ, '_canShowModelAnswer')
+    );
+    if (!isValid) throw new Error('GMCQ - _canShowModelAnswer not found');
+    return true;
+  });
+  checkContent('GMCQ - check _columns attribute', async (content) => {
+    const isValid = GMCQs.every(GMCQ =>
+      _.has(GMCQ, '_columns')
+    );
+    if (!isValid) throw new Error('GMCQ - _columns not found');
+    return true;
+  });
+  updatePlugin('GMCQ - update to v2.0.3', { name: 'adapt-contrib-gmcq', version: '2.0.3', framework: '>=2.0.0' });
+});
+
+describe('GMCQ - v2.0.4 to v2.0.5', async () => {
+  let GMCQs;
+  whereFromPlugin('GMCQ - from v2.0.4', { name: 'adapt-contrib-gmcq', version: '<2.0.5' });
+  whereContent('GMCQ - where GMCQ', async (content) => {
+    GMCQs = content.filter(({ _component }) => _component === 'gmcq');
+    return GMCQs.length;
+  });
+  mutateContent('GMCQ - add _canShowMarking attribute', async (content) => {
+    GMCQs.forEach(GMCQ => {
+      GMCQ._canShowMarking = true;
+    });
+    return true;
+  });
+  checkContent('GMCQ - check _canShowMarking attribute', async (content) => {
+    const isValid = GMCQs.every(GMCQ =>
+      _.has(GMCQ, '_canShowMarking')
+    );
+    if (!isValid) throw new Error('GMCQ - _canShowMarking not found');
+    return true;
+  });
+  updatePlugin('GMCQ - update to v2.0.5', { name: 'adapt-contrib-gmcq', version: '2.0.5', framework: '>=2.0.0' });
```

Comment:
in v2.0.5 framework requirement is bumped to v2.0.11

Reply:
Good spot, listed as 2.0.0 in readme